inspirehep/modules/authors/rest/citations.py

```diff
@@ -92,12 +99,14 @@ def serialize(self, pid, record, links_factory=None):
                 except KeyError:
                     nested_authors = set()
 
-                citation = {}
-                citation['citee'] = {'record': citee}
-                citation['citer'] = {'record': nested_result_source['self']}
-
-                # If at least one author is shared, it's a self-citation.
-                citation['self_citation'] = len(authors & nested_authors) > 0
+                citation = dict(
```

Comment:
Here you're using `dict` constructors to avoid shadowing Python's `id`? In fact, I believe the should be `_id` or `id_`, I don't remember which one the ES Python library is doing.